### PR TITLE
add license to release files

### PR DESCRIPTION
**Pull request by lionello**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -73,6 +83,11 @@ nix:
     repository:
       owner: defang-io
       name: defang
+    post_install: |
```
Comment (edwardrf) [inline .goreleaser.yml:86]: Should these be added to the user's shell init too?

**Review by edwardrf — APPROVED** [review]
LGTM

File: .goreleaser.yml (hunk 1 above)
Comment (lionello) [reply .goreleaser.yml:86]: If I'm not mistaken, the Nix `installShellCompletion` script does that. This section is for Nix only.

**Comment by lionello** [comment]
* the README that gets included is still not great for the binary releases
* running goreleaser outside of the module results in a warning: `reason=not a go module`; not sure what the implications are

**Pull request closed without merging**